owncloudApp/src/main/res/values/strings.xml

```diff
@@ -838,4 +840,8 @@
     <string name="content_description_delete_share">Delete share</string>
     <string name="content_description_file_operations">%1$s operations</string>
 
+    <string name="feedback_dialog_description">Improve the app with your feedback</string>
+    <string name="feedback_dialog_get_in_contact_title">Get in contact</string>
+    <string name="feedback_dialog_get_in_contact_description"><![CDATA[ \n  Ask for help in our <a href=\"%1$s\"><b>forum</b></a> or <a href=\"%2$s\"><b>chat</b></a> with us or contribute on <a href=\"%3$s\"><b>GitHub</b></a>.\n]]></string>
```

Comment:
Is it necessary to have a  `\n` at the beginning?

Reply:
I would also remove the final `.`

Reply:
`\n` don't needed, neither in the back. I thought that it was working, but design is the same.